src/main/java/com/zextras/utils/TranscoderUtils.java

```diff
@@ -0,0 +1,59 @@
+package com.zextras.utils;
+
+import java.nio.charset.StandardCharsets;
+
+/**
+ * Helper util class that performs transcoding operations.
+ */
+public class TranscoderUtils {
```

Comment:
Utility classes should not have public or default constructor, please fix other occurrences too.

Reply:
same, there are no constructors

Reply:
I am referring to the default or implicit public constructor that java adds to classes which do not define at least one explicitly. 
To restrict someone from instantiating such classes, since they only contains static medthods(Utility classes, our case here). 

You can hinder that by adding a private constructor. By adding something like below :)

```
private MyUtilClass() {
    throw new IllegalStateException("Utility class");
}
```